frontend/javascripts/oxalis/view/settings/dataset_settings_view.js

```diff
@@ -74,6 +74,23 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps> {
             <Tag style={{ cursor: "default", marginLeft: 8 }} color={isRGB && "#1890ff"}>
               {isRGB ? "24-bit" : "8-bit"} Layer
             </Tag>
+            {alpha > 0 && isVisible ? (
```

Comment:
The `alpha` check doesn't really make sense right now. The "Make visible" icon is shown but nothing happens when the user clicks it. I think I'd remove the alpha check.